src/org/zaproxy/zap/extension/alert/AlertViewPanel.java

```diff
@@ -155,6 +156,7 @@ private void initialize() {
 	private JScrollPane getAlertPane() {
 		if (alertPane == null) {
 			alertPane = new JScrollPane();
+			alertPane.setHorizontalScrollBarPolicy(ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);

```

Comment:
This is an improvement but there's still an issue, the text areas are allowed to expand but they don't shrink, so if you resize the panel "too much" you might no longer see some of the data after restoring it to the previous size. I guess each text area needs its own scroll pane(?).